hw/dv/tools/dvsim/testplans/sec_cm_fsm_testplan.hjson

```diff
@@ -0,0 +1,28 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+  testpoints: [
+    {
+      name: prim_fsm_check
+      desc: ''' Verify that entering to an undefined state generates a fatal alert.
+
+            Stimulus:
+            - Backdoor force the FSM to any of the undefined values.
+            - Randomly flip the value back to a defined state to ensure the error is latched and
```

Comment:
Is this step needed? I thought a value different from the known states would land it in the default case triggering an alert.